### PR TITLE
Add additional sensors to ecoforest integration.

### PR DESCRIPTION
**Pull request by pjanuario**
## Proposed change

Add additional sensors to ecoforest integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/100681
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ecoforest.markdown [hunk 1]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
 
 ## Sensor
 
-The Ecoforest integration exposes a sensor for the current ambient temperature
+The Ecoforest integration exposes multiple sensors to monitor various features:
+
+- temperature a sensor for the current ambient temperature
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- temperature: a sensor for the current ambient temperature
```

File: source/_integrations/ecoforest.markdown [hunk 2]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
+- cpu_temperature a sensor for the current cpu temperature
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- cpu_temperature: a sensor for the current CPU temperature
```

File: source/_integrations/ecoforest.markdown [hunk 3]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
+- gas_temperature a sensor for the current gas temperature
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- gas_temperature: a sensor for the current gas temperature
```

File: source/_integrations/ecoforest.markdown [hunk 4]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
+- ntc_temperature a sensor for the current ntc probe temperature
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- ntc_temperature: a sensor for the current <abbr title="negative temperature coefficient">NTC</abbr> probe temperature
```

File: source/_integrations/ecoforest.markdown [hunk 5]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
+- status a sensor for the current status of the device
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- status: a sensor for the current status of the device
```

File: source/_integrations/ecoforest.markdown [hunk 6]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
+- alarm a sensor for the current alarm of the device
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- alarm: a sensor for the current alarm of the device
```

File: source/_integrations/ecoforest.markdown [hunk 7]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
 ## Sensor
 
-The Ecoforest integration exposes a sensor for the current ambient temperature
+The Ecoforest integration exposes multiple sensors to monitor various features:
+
+- temperature: a sensor for the current ambient temperature
+- cpu_temperature: a sensor for the current CPU temperature
+- gas_temperature: a sensor for the current gas temperature
+- ntc_temperature: a sensor for the current <abbr title="negative temperature coefficient">NTC</abbr> probe temperature
+- status: a sensor for the current status of the device
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- status: a sensor for the current status of the device. Possible values are: off, starting, pre-heating, on, shutting down, standby, alarm.
```

File: source/_integrations/ecoforest.markdown [hunk 8]
```diff
@@ -44,4 +44,11 @@ The Ecoforest integration exposes a number entity for the device power level.
...
+- alarm: a sensor for the current alarm of the device
```
Comment (c0ffeeca7) [inline source/_integrations/ecoforest.markdown]:
```suggestion
- alarm: a sensor for the current alarm of the device. Possible values are: air depression, pellets, CPU overheating, unknown.
```

Comment (c0ffeeca7) [reply source/_integrations/ecoforest.markdown]: Can we add the possible values? Or is there a particular reason you did not list them?

**Review by frenck — APPROVED** [review]
Thanks all! 👍 

../Frenck

File: source/_integrations/ecoforest.markdown (hunk 8 above)
Comment (pjanuario) [reply source/_integrations/ecoforest.markdown]: we can thanks.

**Pull request merged**